internal/config/main.go

```diff
@@ -97,6 +109,13 @@ func (c *Config) AddFeed(feed Feed) error {
 	return nil
 }
 
+func (c *Config) GetFeeds() []Feed {
```

Comment:
Convention in Go is to have getters as public and values as private. So in this instance we would have:

```go
type Config struct {
  feeds []Feed
  Feeds func() []Feed
}
```

(Will need to update anywhere that references `config.Feeds` to `config.Feeds()`)

See here for more explanation https://go.dev/doc/effective_go#Getters

Reply:
@guyfedwards Thanks! Makes sense. So if that's the case, should I then write up a custom marshaller/unmarshaller for the config to marshal/unmarshal now unexported `feeds` field?

Reply:
Bah, I forgot about that. Whilst I think that would be the more "Go" way to do it, it feels like an unneccessary complexity to add to the codebase vs a clear "GetFeeds" so am happy to leave as is